Pipfile

```diff
@@ -28,10 +28,10 @@ flask-sqlalchemy = "*"
 alembic = "*"
 flask-script = "*"
 flask-migrate = "*"
-kerlescan = {editable = true, git = "https://github.com/RedHatInsights/kerlescan.git", ref = "0.50"}
 bitmath = "*"
 jsonpatch = "*"
 insights-core = "*"
+kerlescan = {editable = true, ref = "0.55", git = "https://github.com/RedHatInsights/kerlescan.git"}
```

Comment:
any reason for changing the order of these fields (editable, ref, git) from (editable, git, ref) previously which was in alphabetical order.

Reply:
I used the `pipenv uninstall kerlescan` command, followed by `pipenv install --editable git+https://github.com/RedHatInsights/kerlescan.git@0.55#egg=kerlescan` command to get pipenv to install the correct version of kerlescan into my env.  Without doing this, regardless of running `pipenv update`, `pipenv lock`, `pipenv install --dev`, etc, running the unit tests would complain about not finding the kerlescan module.
The `pipenv install ...` command put the args in this order and the entry in the position to where it was moved.

Reply:
Interesting, I didn't know about it. Thank you for the explanation.